sw/device/lib/dif/dif_clkmgr_unittest.cc

```diff
@@ -296,5 +296,39 @@ TEST_F(HintableClockTest, GetEnabledError) {
             kDifBadArg);
 }
 
+class MeasureCtrlTest : public ClkMgrTest {};
+
+TEST_F(MeasureCtrlTest, Disable) {
+  EXPECT_WRITE32(CLKMGR_MEASURE_CTRL_REGWEN_REG_OFFSET, 0);
+  EXPECT_EQ(dif_clkmgr_measure_ctrl_disable(&clkmgr_), kDifOk);
+}
+
+TEST_F(MeasureCtrlTest, DisableError) {
+  // Null handle.
```

Comment:
This comment says what, but not why. I don't think it particularly adds much so I'd drop it. (What it's checking is self-evident from the code.)